qml/MonthComponent.qml

```diff
@@ -233,7 +234,7 @@ Item{
             var dayItem = getItemAt(mouse.x, mouse.y)
             var index = getIndexOfChild(monthGrid, dayItem);
             var selectedDate = intern.monthStart.addDays(index);
-            pageStack.push(Qt.resolvedUrl("NewEvent.qml"), {"date":selectedDate, "model":eventModel});
+            bottomEdgeCommit(selectedDate, true);
```

Comment:
Please describe the change in the commit message. It's not clear to me what the old code does, and what it will do after your change. :-)

Reply:
Somehow, i didn't get why, committing the bottom edge as its done for day and week view sets the default calendar in the NewEvent page. I changed the commit message, so that it should be clearer now.

Reply:
I edited the commit message. Is it clear now what the change is about?